Add an intro to the RTE appSettings config (#4672)
umbraco/UmbracoDocs: umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor/configuration.md

Images added through the RTE is by default added to the root of the Media library.

Sometimes you might want to add the images to a specific folder. This folder can be configured using the "Image Upload Folder" setting.

# Advanced Configuration

> The RTE can also accept advanced configuration through the `appSettings.json` file which is fed into the TinyMCE configuration. This is within `Umbraco` / `CMS` / `RichTextEditor` and allows you to configure the following; [Commands](https://www.tiny.cloud/docs/advanced/editor-command-identifiers/#executablecommands), [Plugins](https://www.tiny.cloud/docs/configure/integration-and-setup/#plugins), [CustomConfig](https://www.tiny.cloud/docs/configure/integration-and-setup/), [ValidElements](https://www.tiny.cloud/docs/configure/content-filtering/#valid_elements) and [InvalidElements](https://www.tiny.cloud/docs/configure/content-filtering/#invalid_elements). `CustomConfig` is slightly different to the rest as it extends the core config passed into TinyMCE.

Reviewer: ```suggestion
The RTE can also accept advanced configuration through the `appSettings.json` file fed into the TinyMCE configuration. This is within `Umbraco` / `CMS` / `RichTextEditor` and allows you to configure the following:

* [Commands](https://www.tiny.cloud/docs/advanced/editor-command-identifiers/#executablecommands)
* [Plugins](https://www.tiny.cloud/docs/configure/integration-and-setup/#plugins)
* [CustomConfig](https://www.tiny.cloud/docs/configure/integration-and-setup/)
* [ValidElements](https://www.tiny.cloud/docs/configure/content-filtering/#valid_elements)
* [InvalidElements](https://www.tiny.cloud/docs/configure/content-filtering/#invalid_elements)

`CustomConfig` is slightly different from the rest as it extends the core config passed into TinyMCE.
```

## Allowing script tags in the RTE

> Sometimes you want to allow your content editors to be able to add `<script>` tags into the RTE, this tag is classed as invalid by default so the configuration needs updated to allow it. This is not something that can be done through the CMS and requires advanced configuration in the `appSettings.json` file. This uses the `ValidElements` property to append `,script[*]` to the pre-existing list of valid elements as shown below.

Reviewer: ```suggestion
Sometimes you want to allow your content editors to be able to add `<script>` tags into the RTE. This tag is classed as invalid by default which means the configuration needs to be updated to allow it. This is not something that can be done through the CMS and requires advanced configuration in the `appSettings.json` file. It uses the `ValidElements` property to append `,script[*]` to the pre-existing list of valid elements as shown below.
```

Images added through the RTE is by default added to the root of the Media library.

Sometimes you might want to add the images to a specific folder. This folder can be configured using the "Image Upload Folder" setting.

# Advanced Configuration

> The RTE can also accept advanced configuration through the `appSettings.json` file fed into the TinyMCE configuration. This is within `Umbraco` / `CMS` / `RichTextEditor` and allows you to configure the following:

Reviewer: Updated the formatting for referencing the specific config section. This is how we do it other places in the docs, so though it would make sense to keep it consistent 💪 

```suggestion
The RTE can also accept advanced configuration through the `appSettings.json` file fed into the TinyMCE configuration. This is within `Umbraco:CMS:RichTextEditor` and allows you to configure the following:
```
